rewrite-java/src/main/java/org/openrewrite/java/JavaVisitor.java

```diff
@@ -910,7 +910,9 @@ public J visitWildcard(J.Wildcard wildcard, P p) {
         Space after = visitSpace(right.getAfter(), loc.getAfterLocation(), p);
 
         setCursor(getCursor().getParent());
-
+        if (t == null) {
```

Comment:
if `t == null` then should `visitSpace` be invoked?

Reply:
I really don't think we want to do this, as something systemic is wrong. A `padding` without an element, just doesn't make sense, we really need to prevent a null from ever showing up in a padded element.  It is a non-null value.

Reply:
thanks @tkvangorder  per your comment  the `Space after = visitSpace(right.getAfter(), loc.getAfterLocation(), p);` statement should be moved to the position after the null check block.

Reply:
Hmm I think it could be OK.
We say that the visit methods aren't nullable, but we also say that you should return null from a visit method to remove an element. We embrace this contradiction so as to make the API as user-friendly to recipe authors as possible. The test case showing element removal by returning `null` from a nominally non-null method is a valid way to remove elements. 

That said if `t` is `null` here I don't think there's any point in calling `visitSpace()`. It isn't super harmful to do so, just wastes CPU cycles.

Reply:
thanks @sambsnyd